src/CalcViewModel/Common/CalculatorDisplay.cpp

```diff
@@ -8,25 +8,27 @@
 #include "StandardCalculatorViewModel.h"
 
 using namespace CalculatorApp;
+using namespace CalculatorApp::ViewModel;
+using namespace CalculatorApp::ViewModel::Common;
 using namespace CalculationManager;
 using namespace Platform;
 using namespace std;
 
-CalculatorDisplay::CalculatorDisplay()
+CalculatorApp::ViewModel::Common::CalculatorDisplay::CalculatorDisplay()
```

Comment:
Why not writing all the implementations inside the namespace `CalculatorApp::ViewModel::Common`?

Reply:
done